server/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:10
+
+# If PORT variable is not passed as argument during image build, it defaults to 4000
+# i.e. docker build --build-arg PORT=${ANY PORT} ...
+ARG PORT=4000
+ENV PORT=${PORT}
+
```

Comment:
I believe you also need to specify an `ENV` var for `SEARCH_SERVICE_ENDPOINT`

Reply:
Yeah, that is true. I'll be adding the DB variable as well.